share/runtime-install.tmpl

```diff
@@ -91,14 +86,14 @@ installpkg device-mapper-persistent-data
 installpkg xfsdump
 
 ## needed for LUKS escrow
-installpkg python-volume_key volume_key
-installpkg python-nss
+installpkg volume_key
+installpkg nss-tools
```

Comment:
And rescue. The nss tools only really make sense for the kickstart test (I think they get pulled in anyway through some dep chain but I don't think we really need to keep the /usr/bin parts), but volume_key would be nice to keep around.